W-13484331-apikeysonlyvisibleoncreation-se (#179)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/usermanagement-connect.adoc

> = Connecting to RPA Manager from Salesforce Products

Reviewer: I think the wording "Connecting to RPA Manager" seems incorrect to me. How about - "Connecting RPA Processes from Salesforce Products"? I feel connecting to RPA manager may be interpreted differently by users.

Author: Thank you. That is better.

> RPA Manager requires user API keys for the integration of RPA processes into other Salesforce products. For example, use your API key to configure an RPA connector in a MuleSoft Composer flow.

Reviewer 2: ```suggestion
RPA Manager requires user API keys for the integration of RPA processes into other Salesforce products. For example, you can use your API key to configure an RPA connector in a MuleSoft Composer flow.
```

|===

* You must have an account in RPA Manager to associate your API key with.
* You must have a secure place accessible to store your API key.

[[create-an-api-key-for-a-user]]
== Create a User API Key

Create an API key for yourself to integrate an RPA process into another Salesforce product. After expiration the integrated process instance can no longer connect to RPA Manager from other Salesforce products. Edit the API key to move the expiration date. For security reasons, RPA Manager shows the API Key only during creation. Copy your API Key and save it to a secure place. If you lose an API Key, create a new one.

To create a user API key: 

. Open the *User API Keys* view of the *User Management* module.
. Click *Create*.
. [[form-createuserapikey]] Complete the form *Create User API Key*:
* *User*:
+
Select yourself. Your identity and permissions are used for the connection from another Salesforce product to RPA Manager.
* *Expiration Date*:
+
Select an expiration date for the user API key from the calendar.
. Click *OK*.
. In the window *Key Created!*, click *Copy to clipboard*.
. Copy the API key to a secure place.
. Click *Close*.

> Copy your saved API key into the configuration settings of the Salesforce products you want to integrate the RPA processes.

Reviewer 2: ```suggestion
Copy your saved API key into the configuration settings of the Salesforce products for which you want to integrate the RPA processes.
```

Author: into which?

== Edit a User API Key

Edit a user API key to change its name, description, or expiration date. If you lost access to your API key, create a new one.

To edit a user API key:

. Open the *User API Keys* view of the *User Management* module.
. Click the *Edit* icon image:edit-icon.png["pen-to-paper symbol",1.5%,1.5%] on the panel of the user API key to edit.
. Edit the form *Create User API Key*:
+
For an explanation of the properties, see <<form-createuserapikey, *Create a User API Key*>>.
. Click *Save*.

== Delete a User API Key

> Delete a user API key to revoke REST API access to RPA Manager from the user. Integrated processes using the deleted API key stop working.

Reviewer 2: ```suggestion
Delete a user API key to revoke REST API access from the user to RPA Manager. Integrated processes that use the deleted API key stop working.
```
